### PR TITLE
Added packages to list of contributors.

### PR DESCRIPTION
**Pull request by peterfabian**
Include all the contributors, also to sub packages in the contributor list. 
Also, changed the format to HTML to make it easier to use in block editor and switched to use the default branch, for now, and for simplicity.

To test: 

1. create a GH oAuth token via https://github.com/settings/tokens, it needs access to the repo
2. run `grunt contributors`
3. add date (e.g. 2020-03-10)
4. add GH token
5. Nonempty output should be generated into contributors.html

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -215,11 +215,33 @@ module.exports = function( grunt ) {
...
+					'echo "<h2>WooCommerce core</h2>" >> contributors.html',
+					'echo "Generating contributor list for WC core since <%= fromDate %>"',
 					'./node_modules/.bin/githubcontrib --owner woocommerce --repo woocommerce --fromDate <%= fromDate %>' +
-					' --authToken <%= authToken %> --cols 6 --sortBy contributions --format md --sortOrder desc' +
-					' --showlogin true --sha <%= sha %> --filter renovate-bot > contributors.md'
-				].join( '&&' )
+					' --authToken <%= authToken %> --cols 6 --sortBy contributions --format html --sortOrder desc' +
+					' --showlogin true --filter "renovate-bot,apps/renovate,renovate,renovate[bot]" >> contributors.html',
+					'echo "Output generated to contributors.html."',
+				].join('&&')
```
Comment (claudiosanches) [inline Gruntfile.js:244]: Coding standards requires spaces in `.join( '&&' )`

File: Gruntfile.js [hunk 2]
```diff
@@ -215,11 +215,33 @@ module.exports = function( grunt ) {
...
+					'echo "Generating contributor list for Action Scheduler since <%= fromDate %>"',
+					'./node_modules/.bin/githubcontrib --owner woocommerce --repo action-scheduler --fromDate <%= fromDate %>' +
+					' --authToken <%= authToken %> --cols 6 --sortBy contributions --format html --sortOrder desc' +
+					' --showlogin true --filter "renovate-bot,apps/renovate,renovate,renovate[bot]" >> contributors.html',
+					'echo "<h2>REST API</h2>" >> contributors.html',
+					'echo "Generating contributor list for REST API since <%= fromDate %>"',
+					'./node_modules/.bin/githubcontrib --owner woocommerce --repo woocommerce-rest-api --fromDate <%= fromDate %>' +
+					' --authToken <%= authToken %> --cols 6 --sortBy contributions --format html --sortOrder desc' +
+					' --showlogin true --filter "renovate-bot,apps/renovate,renovate,renovate[bot]" >> contributors.html',
+					'echo "<h2>WooCommerce core</h2>" >> contributors.html',
```
Comment (claudiosanches) [inline Gruntfile.js:238]: I think we should move core to the first, like we does with the changelogs.

**Review by claudiosanches — CHANGES_REQUESTED** [review]
It works fine, but feels weird having this script here, currently some grunt packages are installed only to allow run this script once we release something.
We could move it to a new repository and create a proper tool for it, so other teams can use too.
Or at least move it to the `bin/` directory, so `grunt-prompt` can get removed, and also `grunt-shell`, since we run e2e tests now by the `package.json` file, those script we have on Grunt can get removed too.

But if we plan to keep on core, at least we should include an entry about `contributors.html` in the `.gitignore`.

contributors.html

File: Gruntfile.js (hunk 2 above)
Comment (peterfabian) [reply Gruntfile.js:238]: I wanted to politely give credit to our guests first, but we can swap, sure.

**Comment by peterfabian** [comment]
> It works fine, but feels weird having this script here, currently some grunt packages are installed only to allow run this script once we release something.
We could move it to a new repository and create a proper tool for it, so other teams can use too.

Fair enough, but this is basically just a call of a npm package, so I'm not sure it deserves its own repository. It's true, though, that in some way this could be included in an 'ideal repo' implementation we will create.

> Or at least move it to the bin/ directory, so grunt-prompt can get removed, and also grunt-shell, since we run e2e tests now by the package.json file, those script we have on Grunt can get removed too.

Do you mean creating a bash script and placing it into bin folder instead of running the command as a grunt script? Sounds like a good idea, as grunt anyway just runs shell commands. Please check the update.

> But if we plan to keep on core, at least we should include an entry about contributors.html in the .gitignore.

Done that, thanks for noticing.

**Review by claudiosanches — APPROVED** [review]
Great work!

**Pull request merged**